src/Form/Formbot.js

```diff
@@ -132,7 +148,11 @@ export default class Formbot extends React.Component {
           });
         }
       } catch (err) {
-        errorMsg = err.message;
+        if (fromSchema) {
+          errorMsg = err ? err.errors[0] : undefined;
```

Comment:
Is it possible for `err` to be falsy, if we're inside catch?

Reply:
just porting it over from the EMR but yeah probably not